webpack.common.js

```diff
@@ -152,12 +152,18 @@ const modernConfig = {
 // Common module exports
 // noinspection WebpackConfigHighlighting
 module.exports = {
-    'legacyConfig': merge(
-        legacyConfig,
+    'modernConfig': merge.strategy({
```

Comment:
Should this be 'legacyConfig'?

Reply:
Yes, of course. I pushed a fix commit and can amend it to the first one if you'd like.